### ISSUE TITLE
Windows Build

### ISSUE THREAD

**Issue by flashpixx** [tensorflow/io#1832]
Hello,

I’m using in a Python 3.10 project tensorflow-io-gcs-filesystem 0.33 as transitive dependency of a library. As build tool I’m using Poetry. Under Linux / OSX everything workfs fine, but I try to build / run my tool under Windows (I’m using Gitlabs Windows Shared Runner).

During “poetry install” the installation breaks:

```
Installing tensorflow-io-gcs-filesystem (0.33.0)
RuntimeError
Unable to find installation candidates for tensorflow-io-gcs-filesystem (0.33.0)
C:\Python310\lib\site-packages\poetry\installation\chooser.py:76 in choose_for
72|
73| links.append(link)
74|
75| if not links:
76| raise RuntimeError(f"Unable to find installation candidates for {package}")
77|
78| # Get the best link
79| chosen = max(links, key=lambda link: self._sort_key(package, link))
```

How can I build my system on Windows / install the correct dependency?
Thanks a lot

**Comment by egerlach** [comment]
Is this a dupe of #1815?

**Comment by RRiva** [comment]
Yes it is. See also the discussion at https://github.com/python-poetry/poetry/issues/8271

**Comment by mraunak** [comment]
Hi @flashpixx can you please try the solution provided in https://github.com/tensorflow/tensorflow/issues/58674#issuecomment-1593891706
Please let me know if this doesn't resolve your issue
